docs/tutorials/get-started-with-ngo.md

```diff
@@ -104,6 +104,26 @@ This section guides you through creating an object that spawns for each connecte
 
 12. Save the scene by pressing **Ctrl/Cmd** + **S** (selecting **File** > **Save**).
 
+## Test starting a host in Unity Editor
+
+To ensure **NetworkManager** and **PlayerPrefab** are functioning correctly, test starting a host in the Unity Editor. A host plays the role of a server and a client at the same time.
+
+You can test your Hello World project through the Unity Editor or a command line helper. If you choose the latter, refer to [Create a command line helper](../tutorials/command-line-helper/). Otherwise, refer to the following instructions to test through the Unity Editor. Only the Plane appears on the server until the first client connects. Then, Netcode for GameObjects spawns a new Player prefab for each connected client; however, they overlap in the Game view.
```

Comment:
```suggestion
You can test your Hello World project using the Unity Editor or a command-line helper. If you choose the latter, refer to [Create a command line helper](../tutorials/command-line-helper/). Otherwise, refer to the following instructions to test using the Unity Editor. Only the Plane appears on the server until the first client connects. Then, Netcode for GameObjects spawns a new Player prefab for each connected client; however, they overlap in the Game view.
```